ansible/roles/supervisord/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: restart supervisord
+  service: name=supervisord state=restarted
```

Comment:
This should eventually add a handler to just restart the app not supervisor itself. Theres also a way to reload configs but this is ok for now.